bitnami/haproxy/README.md

```diff
@@ -0,0 +1,408 @@
+| `nodeAffinityPreset.key`             | Node label key to match. Ignored if `affinity` is set                                     | `""`                    |
+| `nodeAffinityPreset.values`          | Node label values to match. Ignored if `affinity` is set                                  | `[]`                    |
+| `configuration`                      | haproxy configuration                                                                     | `<dummy configuration>` |
+| `containerPorts`                     | List of container ports to enable in the haproxy container                                | `[]`                    |
+| `existingConfigmap`                  | configmap with HAProxy configuration                                                      | `""`                    |
+| `affinity`                           | Affinity for haproxy pods assignment                                                      | `{}`                    |
+| `nodeSelector`                       | Node labels for haproxy pods assignment                                                   | `{}`                    |
+| `tolerations`                        | Tolerations for haproxy pods assignment                                                   | `[]`                    |
+| `updateStrategy.type`                | haproxy statefulset strategy type                                                         | `RollingUpdate`         |
+| `priorityClassName`                  | haproxy pods' priorityClassName                                                           | `""`                    |
+| `lifecycleHooks`                     | for the haproxy container(s) to automate configuration before or after startup            | `{}`                    |
+| `extraEnvVars`                       | Array with extra environment variables to add to haproxy nodes                            | `[]`                    |
+| `extraEnvVarsCM`                     | Name of existing ConfigMap containing extra env vars for haproxy nodes                    | `nil`                   |
+| `extraEnvVarsSecret`                 | Name of existing Secret containing extra env vars for haproxy nodes                       | `nil`                   |
+| `extraVolumes`                       | Optionally specify extra list of additional volumes for the haproxy pod(s)                | `[]`                    |
+| `extraVolumeMounts`                  | Optionally specify extra list of additional volumeMounts for the haproxy container(s)     | `[]`                    |
+| `sidecars`                           | Add additional sidecar containers to the haproxy pod(s)                                   | `[]`                    |
+| `initContainers`                     | Add additional init containers to the haproxy pod(s)                                      | `{}`                    |
```

Comment:
I see that many of these commands were already defined below. It could be some issues using `helm-readme-generator`, but could you fix it?